Cargo.toml

```diff
@@ -49,9 +49,9 @@ nix = "0.20.0"
 openat = "0.1.21"
 openat-ext = "^0.2.0"
 os-release = "0.1.0"
-ostree = { version = "0.9.1", features = ["v2020_4"] }
+ostree = { version = "0.11.0", features = ["v2021_1"] }
```

Comment:
Shouldn't this be v2021_2? I guess we're not using any features from that version just yet.